tests/kola/rpm-ostree/replace-rt-kernel

```diff
@@ -5,7 +5,9 @@
 ##   # We've seen some OOM when 1024M is used in similar tests:
 ##   # https://github.com/coreos/fedora-coreos-tracker/issues/1506
 ##   minMemory: 2048
-##   description: Verify replacing the current kernel with an 
+##   # RT/NFV packages are only x86_64 on the centos stream repos
+##   architectures: x86_64
+##   description: Verify replacing the current kernel with an
```

Comment:
hmm.. looking at the code below I see there is already a `case $basearch in` for the RT kernel test. The basic kernel replacement should work.

Maybe we just disable the nfv repo in the config before the first test and then re-enable it for the RT test.

Reply:
updated.